src/Algebra/Construct/Flip/Op.agda

```diff
@@ -0,0 +1,241 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Flipping the arguments of a binary operation preserves many of its
+-- algebraic properties.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Construct.Flip.Op where
+
+open import Algebra
+open import Data.Product using (swap)
+import Data.Sum as Sum
+open import Function.Base using (flip)
+open import Level using (Level)
+open import Relation.Binary.Core using (Rel; _Preserves₂_⟶_⟶_)
+open import Relation.Binary.Definitions using (Symmetric)
+
+private
+  variable
+    a b p ℓ ℓ₁ ℓ₂ : Level
+    A B : Set a
+    ∼ ≈ ≋ : Rel A ℓ
```

Comment:
`∼` and `≋` are pretty unfamiliar symbols to use in the algebra hierarchy, so I'd prefer if they weren't made variables. Indeed they're only used in the `preserves₂` proof, so perhaps can we just make them implicit parameters of that proof instead?

Reply:
Yup, you're right. There was some other garbage in that variable declaration too, sorry. I cleaned it up now.